modAionImpl/src/org/aion/zero/impl/sync/TaskImportBlocks.java

```diff
@@ -581,7 +583,7 @@ private ImportResult importBlock(AionBlock b, String displayId, PeerState state)
                 log.info("Compacting state database due to slow IO time.");
             }
             t1 = System.currentTimeMillis();
-            //this.chain.compactState();
```

Comment:
I think this is a merge bug. It should be uncommented.

Reply:
The change came from outside this PR, but can be solved here.

Reply:
this is not a merge bug. I commented before the 0.3.2.1 release. The compact states should check the config settings before execute it.